code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/gun
 	name = "energy gun"
-	desc = "Another bestseller of Lawson Arms and the FTU, the LAEP90 Perun is a versatile energy based sidearm, capable of switching between low and high capacity projectile settings. In other words: Stun or Kill."
+	desc = "Another bestseller of Lawson Arms and"+TSC_HEPH+", the LAEP90 Perun is a versatile energy based sidearm, capable of switching between low and high capacity projectile settings. In other words: Stun or Kill."

```

Comment:
You'll need a space between the ``and"`` and ``+TSC_HEPH``, or else it'll say ``Another bestsellar of Lawson Arms andHephethsus`` or however it's actually spelled (yay for that constant)

Reply:
Apparently you already got it so wew.